back_office/back_office_home_page/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BackofficeHomePageConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'back_office.back_office_home_page'
```

Comment:
The app is already in the back_office folder, the app should be named home_page, but be created inside of the back_office folder so that the app name would be back_office.home_page

Create a new app with correct name, move the code to the new app, delete this one.